### PR TITLE
Readme file update

### PR DESCRIPTION
**Pull request by tnboyko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-this repo is for CI/CD demonstration 
\ No newline at end of file
+this repo is for CI/CD demonstration 
+
+students of qa_oct23 are the best
```
Comment (tnboyko) [inline README.md]:
```suggestion
students of qa_oct23 are the best
```

no

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-this repo is for CI/CD demonstration 
\ No newline at end of file
+this repo is for CI/CD demonstration
```
Comment (tnboyko) [inline README.md:1]: let's change this title to another one

File: README.md (hunk 1 above)
Comment (djtyrf312) [inline README.md]:
```suggestion
students and coach and mentors of qa_oct23 are the best
```

**Review by djtyrf312 — COMMENTED** [review]
🔥

**Pull request merged**